GiEnJul/appsettings.json

```diff
@@ -7,5 +7,19 @@
     }
   },
   "AllowedHosts": "*",
-  "TableConnectionString": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;"
+  "TableConnectionString": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;QueueEndpoint=http://127.0.0.1:10001/devstoreaccount1;TableEndpoint=http://127.0.0.1:10002/devstoreaccount1;",
+  "MailSettings": {
+    "Mail": "noreply.gienjul.testz@gmail.com",
+    "DisplayName": "Gi En Jul",
+    "Password": "kulmailkonto2021",
+    "Host": "smtp.gmail.com",
+    "Port": 587
+  },
+  "OneMailSettings": {
+    "Mail": "noreply@gienjul.no",
+    "DisplayName": "Gi En Jul",
+    "Password": "password",
+    "Host": "send.one.com",
+    "Port": 465
```

Comment:
I think there should only be one MailSettings section.
The commit should include local settings, that can be replaced in the deploy pipeline.
We could consider using Secrets in AzureKeyVault for the passwords.

Maybe make a note of the other settings in another issue?

Reply:
Created #241. The current mail should probably not be used in the future. Maybe prod and dev could use the same mail(noreply@gienjul.no)?